cli/asc.js

```diff
@@ -643,6 +643,17 @@ exports.main = function main(argv, options, callback) {
     const passes = [];
     function add(pass) { passes.push(pass); }
 
+    // enchance options for inline passes
+    if (optimizeLevel >= 2 && shrinkLevel === 0) {
+      module.setAlwaysInlineMaxSize(12);
+      module.setFlexibleInlineMaxSize(60);
+      module.setOneCallerInlineMaxSize(60);
```

Comment:
What would be the downside of making `OneCallerInlineMaxSize` basically unlimited? Just curious :)

Reply:
Infinite code size? =) Basically it significantly increase build speed. Also it could even slowdown code due to inline huge functions will increase register pressure and stack usage